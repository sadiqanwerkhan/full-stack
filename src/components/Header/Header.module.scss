@import '../../styles/variables.scss';

.header {
    width: 100%;
    background-color: $bodyBg;
    padding: $spacing-kilo $spacing-mega;
    border-bottom: 1px solid $n300;
    line-height: 0;
}