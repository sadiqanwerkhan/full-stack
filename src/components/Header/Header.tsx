import React, { memo, FC } from "react";
import Image from "next/image";
import styles from "./Header.module.scss";

const Header: FC = () => (
  <header className={styles.header}>
    <Image src="/logo.svg" width={125} height={37} alt="SumUp Logo" />
  </header>
);

Header.displayName = "Header";

export { Header };
