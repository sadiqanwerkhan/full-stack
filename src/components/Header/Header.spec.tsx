import "@testing-library/jest-dom";

import React from "react";
import { render } from "@testing-library/react";

import { Header } from "./Header";

describe("Header", () => {
  it("should render matching default styles", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
