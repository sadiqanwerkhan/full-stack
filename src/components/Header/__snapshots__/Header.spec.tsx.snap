// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render matching default styles 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <img
      alt="SumUp Logo"
      data-nimg="1"
      decoding="async"
      height="37"
      loading="lazy"
      src="/logo.svg"
      style="color: transparent;"
      width="125"
    />
  </header>
</DocumentFragment>
`;
